projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -676,7 +676,7 @@ function wpcom_launchpad_get_task_definitions() {
 			// We do not want this mapped to the 'subscribers_added' task, since this task supports
 			// being marked as complete in situations where subscribers are not added.
 			'get_title'            => function () {
-				return __( 'Add your first subscribers', 'jetpack-mu-wpcom' );
+				return __( 'Add subscribers', 'jetpack-mu-wpcom' );
```

Comment:
We already have [a translation for Add subscribers](https://translate.wordpress.com/projects/wpcom/es/default?filters%5Bterm%5D=Add+subscribers&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filters%5Bview%5D=&filter=Filter&sort%5Bby%5D=priority&sort%5Bhow%5D=desc).